### PR TITLE
[stable/nginx-ingress] Make nginx-ingress hpa resource configurable

### PR DESCRIPTION
**Pull request by okgolove**
Signed-off-by: Mikhail Naletov <okgolove@markeloff.net>

#### What this PR does / why we need it:
Sometimes you may want to specify only CPU or memory as autoscaling metric.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by ChiefAlexander** [comment]
Hmmm. This would be a breaking change to anyone who is using the defaults provided. I am fine with this change but we have to keep the default values in place to avoid a breaking change. For example if a user just used `--set autoscaling.enabled=true` when deploying. Upgrading now would cause their autoscaling settings to change to nil. Thoughts?

**Comment by okgolove** [comment]
@ChiefAlexander I am totally agree.
We can return the default values and in case of you want to disable (memory, for example) autoscaling metric you can set it null:
```targetCPUUtilizationPercentage: null```

Is it ok?

**Comment by ChiefAlexander** [comment]
That sounds great!

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.11.5
+version: 1.11.6
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: Another change got in before yours. This will need to be bumped to 1.12.1

**Comment by ChiefAlexander** [comment]
Thanks!
/lgtm

**Pull request merged**